### DISCUSSION TITLE
Custom Cateories

### DISCUSSION THREAD

**Discussion by Tullstar** [mediacms-io/mediacms#481] [Q&A]
We host audio and video files that we have recorded and I like this program but have a few things that I need that I'm not sure are implemented:

1. Custom Categories - I really need to be able group / sort by events and years so a custom and hopefully nested category system. So Dallas -> 2021, Dallas -> 2022, Austin -> 2021,  etc.  Is that possible?

2. Author/Speaker - I would post videos but wish I could list an author or speaker name that could also be sorted by.

Hope that makes sense, Those are two main hurdles to being able to us this for my project. Thanks for any help.

**Comment by mgogoulos** [comment]
Hi, 

you can add categories (from the admin dashboard only) and they will appear on the listing, you may add a parent category as well but that relationship will not show up. It shouldn't be hard to add but it's not there yet

Not sure about the second, you can have listing per user and there's a Member's page (link on the left sidebar)...
